arches/app/tasks.py

```diff
@@ -315,6 +315,28 @@ def edit_bulk_string_data(userid, load_id, module_id, graph_id, node_id, operati
         user = User.objects.get(id=userid)
         notify_completion(msg, user)
 
+@shared_task
+def bulk_data_deletion(userid, load_id, graph_id, nodegroup_id, resourceids):
+    from arches.app.etl_modules import bulk_data_deletion
+
+    logger = logging.getLogger(__name__)
+
+    try:
+        BulkDataDeletion = bulk_data_deletion.BulkDataDeletion(loadid=load_id)
+        BulkDataDeletion.run_load_task(userid, load_id, graph_id, nodegroup_id, resourceids)
```

Comment:
If changed to run_bulk_task in the module, it would need to be updated here as well.